src/main/java/com/yandex/money/api/methods/params/P2pTransferParams.java

```diff
@@ -125,18 +125,23 @@ public P2pTransferParams build() {
 
         private Map<String, String> makeParams() {
             HashMap<String, String> params = new HashMap<>();
+
             params.put(Params.TO.paramName, to);
             params.put(Params.COMMENT.paramName, comment);
             params.put(Params.MESSAGE.paramName, message);
             params.put(Params.LABEL.paramName, label);
-            params.put(Params.CODEPRO.paramName, codepro.toString());
-            if (expirePeriod != null) {
-                params.put(Params.EXPIRE.paramName, expirePeriod.toString());
-            }
+
+            params.put(Params.CODEPRO.paramName, getStringOrNull(codepro));
+            params.put(Params.EXPIRE.paramName, getStringOrNull(expirePeriod));
```

Comment:
I don't like here that you include parameters that had to be omitted. If something will change this could be a cause of a problem.